tsdb/head.go

```diff
@@ -213,6 +213,32 @@ func NewHead(r prometheus.Registerer, l log.Logger, wal *wal.WAL, opts *HeadOpti
 	return h, nil
 }
 
+func (h *Head) CheckOutOfOrderSample(ref uint64, t int64, v float64) bool {
```

Comment:
This method needs a comment, also stating that it is not used by Prometheus itself. On another note, do we also need to pass labels to it in case the caller does not have a reference yet?

PS: @bboreham does this method help the use case?

Reply:
I have edited my above comment, the function is doing something else than what was intended.